package.json

```diff
@@ -18,13 +18,15 @@
   },
   "devDependencies": {
     "@11ty/eleventy": "0.11.0",
+    "babel-eslint": "^10.1.0",
```

Comment:
Let's use a exact version

Reply:
I'm curious, why do we need this dependency? We don't use babel.

Reply:
It allowed the arrow function syntax to work. We probably don't need it now that I'm thinking about it. I probably need to just add a rule in the `.eslint` config.

Reply:
What about changing the parser like I did in the previous example here https://github.com/ideas42ventures/ideas42ventures.com/pull/88/files#diff-6884918dc8291219be508e05e28965b958c734def85324f3b53858ea4702090fR6 That fixed that issue there. No other deps needed.